doc/authors.md

```diff
@@ -1,36 +1,45 @@
 Zcash Contributors
 ==================
 
-Jack Grigg (323)
-Simon Liu (220)
-Sean Bowe (180)
+isle2983 (2983)
```

Comment:
@daira Can you create a PR or just copy the patch here and I can include as part of this PR.  Since the fix is release process related I think it would be ok to include the patch commit here.